Intégration du nouveau texte de tutoriel.

edemartel opened the pull request.

edemartel committed:
Intégration du nouveau texte de tutoriel.

JesseEmond commented on src/components/Tutorial.vue:
```diff
@@ -65,34 +65,82 @@ export default defineComponent({
                 // depend on loaded data for its placement (or e.g. no target).
                 {
-                    content: this.$t('tutorial_welcome_html'),
+                    content: this.$t('tutorial_step_1'),
```
nit: perso j'avais essayé d'avoir l'ordre du step découplé du nom, comme ça l'ordering de tutoriel est contrôlé purement par ce qui est dans `Tutorial.vue` (même raison que je ne mettais pas `"data-tutorial-step="1"` mettons).

Qu'est-ce que tu en penses? Ausi je mettais `_html` dans le nom pcq c'est un peu spécial que ça soit du HTML pris direct de i18n (avec comment `content` fonctionne pour `v3-tour`)

edemartel replied:
D'accord pour les noms, mais pas pour le HTML. On s'en fout que ça contiennent du HTML, ça reste du contenu pareil.

JesseEmond commented on src/locales/fr-CA.ts:
```diff
@@ -193,8 +193,14 @@ export default {
+    tutorial_step_7: "<p>Nombre d'événements extrêmes survenus dans la circonscription et l'année sélectionnées.</p><p>Vous pouvez sélectionner quels types d'événements extrêmes afficher (ici un total de 3 événements extrêmes, dont 2 tornades et 1 vague de chaleur en 2022)</p>",
+    tutorial_step_8: "<p>Certaines années contiennent des <b>faits saillants</b>. Cliquez sur l'icône pour en savoir plus.</p>",
+    tutorial_step_9: "<p>Vous êtes préoccupé(e)s? Contactez vos candidat(e)s aux prochaines élections.</p>",    
+    tutorial_step_10: "<p>Pour revoir ce tutoriel, ou pour de plus amples informations, consultez <b>l'aide</b>!</p>"
```
nit: `l'<b>aide</b>` pour pas que le `l` soit orange? (e.g. on fait ça pour `année` aussi)

JesseEmond commented on src/locales/fr-CA.ts:
```diff
@@ -193,8 +193,14 @@ export default {
+    tutorial_step_4: "<p>Voici la température pour l'<b>année</b> et la <b>région</b> sélectionnées (ici 2022 pour la province de Québec).</p><p>Remarquez les changements de température par rapport à 1990 (ici +2,2 °C).</p>",
+    tutorial_step_5: "<p>Vous pouvez changer l'<b>année</b> pour explorer les données de températures moyennes observées par le passé et les projections futures ainsi que les événements climatiques extrêmes survenus.</p>",
+    tutorial_step_6: "<p>Sélectionnez la circonscription à afficher ou l'ensemble du Québec</p>",
+    tutorial_step_7: "<p>Nombre d'événements extrêmes survenus dans la circonscription et l'année sélectionnées.</p><p>Vous pouvez sélectionner quels types d'événements extrêmes afficher (ici un total de 3 événements extrêmes, dont 2 tornades et 1 vague de chaleur en 2022)</p>",
```
on pourrait mettre `3 événements extrêmes` en orange ici peut-etre

JesseEmond commented on src/locales/fr-CA.ts:
```diff
@@ -193,8 +193,14 @@ export default {
+    tutorial_step_6: "<p>Sélectionnez la circonscription à afficher ou l'ensemble du Québec</p>",
+    tutorial_step_7: "<p>Nombre d'événements extrêmes survenus dans la circonscription et l'année sélectionnées.</p><p>Vous pouvez sélectionner quels types d'événements extrêmes afficher (ici un total de 3 événements extrêmes, dont 2 tornades et 1 vague de chaleur en 2022)</p>",
+    tutorial_step_8: "<p>Certaines années contiennent des <b>faits saillants</b>. Cliquez sur l'icône pour en savoir plus.</p>",
+    tutorial_step_9: "<p>Vous êtes préoccupé(e)s? Contactez vos candidat(e)s aux prochaines élections.</p>",    
```
Peut-etre `Contactez vos candidat(e)s` en orange? C'est comme tu veux

edemartel committed:
Modifications au texte du tutoriel

Utilisation de noms plus indicatifs pour les clés de localisation
pour les étapes, et utilisation de gris foncé plutôt qu'orange
pour les accents du tutoriel pour éviter que ça ressemble à des liens.
